web/src/main/webapp/dataViews/heavyHittersView.ts

```diff
@@ -170,16 +176,31 @@ export class HeavyHittersView extends RemoteTableObjectView {
   */
 export class HeavyHittersReceiver2 extends OnCompleteRenderer<TopList> {
     public constructor(public hhv: HeavyHittersView,
-                       protected originalTableId: RemoteObjectId,
                        public operation: ICancellable) {
         super(hhv.page, operation, "Heavy hitters -- exact counts");
     }
 
     run(newData: TopList): void {
-        let newPage = new FullPage("Heavy hitters", "HeavyHitters", this.hhv.page);
-        let newHhv = new HeavyHittersView(newData, newPage, this.hhv.tv, this.hhv.schema, this.hhv.order, false);
-        newPage.setDataView(newHhv);
-        this.page.insertAfterMe(newPage);
+        let newHhv = new HeavyHittersView(newData, this.page, this.hhv.tv, this.hhv.schema, this.hhv.order, false);
+        this.page.setDataView(newHhv);
         newHhv.fill(newData.top, this.elapsedMilliseconds());
     }
 }
+
+export class HeavyHittersReceiver3 extends OnCompleteRenderer<TopList> {
```

Comment:
Maybe you can add a little comment explaining why you need this class. I.e. who is using it.